client/src/document/on-github.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+
+import { Doc } from "./types";
+
+export function OnGitHubLink({ doc }: { doc: Doc }) {
+  return (
+    <div id="on-github">
```

Comment:
```suggestion
    <div id="on-github" className="report-problem">
```

Reply:
Can't you use the `#on-github`?

Reply:
Nope, ids in CSS is evil 😄

Reply:
Added a classname. But I prefer `on-github` because it's not always an action.